### PR TITLE
fix(popover): remove duplicate calculation to prevent popover from covering trigger

### PR DESCRIPTION
**Pull request by liamdebeasi**
Issue number: N/A

---------

## What is the current behavior?

As noted in https://github.com/ionic-team/ionic-framework/issues/28119, we are doing additional work when computing the popover positioning that doesn't need to happen.  When computing `triggerTop` we add the trigger height to the top coordinate of the trigger: https://github.com/ionic-team/ionic-framework/blob/ac2c8e6c22da4d0d8224def24ddef56ee9d26246/core/src/components/popover/utils.ts#L834

This is wrong because that would actually give us the bottom coordinate of the trigger. However, we then subtract `triggerHeight` from `triggerTop` to get the coordinate when computing the adjusted `top` value: https://github.com/ionic-team/ionic-framework/blob/ac2c8e6c22da4d0d8224def24ddef56ee9d26246/core/src/components/popover/utils.ts#L877

However, there are edge cases where that value is not negated (i.e. that code is not run) so it can cause layout issues.

## What is the new behavior?

- Removed the additional `triggerHeight` computation

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

## Other information

Note: The screenshot updates here are correct. The popover should typically never cover the trigger element. Instead, the popover (correctly) is displayed above the trigger. This allows the full popover to be displayed relative to the trigger without covering it up.

### CODE REVIEW

File: core/src/components/datetime/test/position/datetime.e2e.ts-snapshots/datetime-position-popover-ios-ltr-Mobile-Safari-linux.png
Comment (amandaejohnston) [inline core/src/components/datetime/test/position/datetime.e2e.ts-snapshots/datetime-position-popover-ios-ltr-Mobile-Safari-linux.png:1]: I know you said the screenshot updates are correct, but they seem to show the opposite from what you described: the popover used to sit above the trigger, but now it's covering the trigger up slightly. Is that intentional? 🤔

File: core/src/components/popover/utils.ts [hunk 1]
```diff
@@ -888,6 +886,8 @@ export const calculateWindowAdjustment = (
     }
   }
 
+  console.log(top, arrowTop);
```
[the remark this answers is not in the document]
Comment (liamdebeasi) [reply core/src/components/popover/utils.ts]: Nope fixed in https://github.com/ionic-team/ionic-framework/pull/28262/commits/4cd23fb1dab86cee3d0b858e52d0e63b2fe3620a

File: core/src/components/datetime/test/position/datetime.e2e.ts-snapshots/datetime-position-popover-ios-ltr-Mobile-Safari-linux.png
Comment (liamdebeasi) [reply core/src/components/datetime/test/position/datetime.e2e.ts-snapshots/datetime-position-popover-ios-ltr-Mobile-Safari-linux.png:1]: Oh oops I got confused 🤔 I'm going to move this back to draft until I can look at this some more.

**Comment by liamdebeasi** [comment]
As Amanda noted, popover does appear to be working correctly. I got confused during my initial analysis. Going to close.

**Pull request closed without merging**